dts/arm/microchip/pic32cmmc.dtsi

```diff
@@ -97,6 +97,50 @@
 				write-block-size = <4>;
 			};
 		};
+
+		sercom0: sercom@42000400 {
+			compatible = "atmel,sam0-sercom";
+			reg = <0x42000400 0x31>;
+			interrupts = <9 0>;
+			clocks = <&gclk 19>, <&mclk 0x1c 1>;
+			clock-names = "GCLK", "MCLK";
+			atmel,assigned-clocks = <&gclk 0>;
+			atmel,assigned-clock-names = "GCLK";
+			status = "disabled";
+		};
+
+		sercom1: sercom@42000800 {
```

Comment:
why is no board added? None of this code is tested in CI

Reply:
Do I need to include the board for this PR?